### PR TITLE
Add basic endpoint for platform docs

### PR DESCRIPTION
**Pull request by dcramer**
http://localhost:8000/api/0/projects/sentry/internal/docs/python-flask/

```
{
    "isFramework": true,
    "html": "<div class=\"section\" id=\"installation\">\n<h2>Installation</h2>\n<p>If you haven&#8217;t already, install raven with its explicit Flask dependencies:</p>\n<div class=\"highlight-python\"><div class=\"highlight\" style=\"background: #ffffff\"><pre style=\"line-height: 125%\">pip install raven[flask]\n</pre></div>\n</div>\n</div>\n\n\n<div class=\"section\" id=\"setup\">\n<h2>Setup</h2>\n<p>The first thing you&#8217;ll need to do is to initialize Raven under your application:</p>\n<div class=\"highlight-python\"><div class=\"highlight\" style=\"background: #ffffff\"><pre style=\"line-height: 125%\"><span style=\"color: #2eb0f7\">from</span> <span style=\"color: #111111\">raven.contrib.flask</span> <span style=\"color: #2eb0f7\">import</span> <span style=\"color: #111111\">Sentry</span>\n<span style=\"color: #111111\">sentry</span> <span style=\"color: #4b4f5c\">=</span> <span style=\"color: #111111\">Sentry(app,</span> <span style=\"color: #111111\">dsn</span><span style=\"color: #4b4f5c\">=</span><span style=\"color: #e8535a\">&#39;___DSN___&#39;</span><span style=\"color: #111111\">)</span>\n</pre></div>\n</div>\n<p>If you don&#8217;t specify the <code class=\"docutils literal\"><span class=\"pre\">dsn</span></code> value, we will attempt to read it from\nyour environment under the <code class=\"docutils literal\"><span class=\"pre\">SENTRY_DSN</span></code> key.</p>\n</div>\n",
    "link": "https://docs.getsentry.com/hosted/clients/python/integrations/flask",
    "name": "Flask",
    "sdk": "raven-python"
}
```

### CODE REVIEW

File: src/sentry/api/endpoints/project_platform_docs.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        result = cache.get(cache_key)
+        if result is None:
+            session = http.build_session()
+            result = session.get(DOC_URL.format(platform=platform)).json()
+            cache.set(cache_key, result, 3600)
+
+        return Response({
+            'name': result['name'],
+            'html': result['body'],
+            'sdk': result['client_lib'],
```
Comment (mattrobenolt) [inline src/sentry/api/endpoints/project_platform_docs.py]: Is there a reason we're transforming these keys?

Wouldn't we ideally just do: `return Response(result)`

Comment (dcramer) [inline src/sentry/api/endpoints/project_platform_docs.py]: Because APIs should be stable and they are not formatted to our spec.

Comment (mattrobenolt) [inline src/sentry/api/endpoints/project_platform_docs.py]: Yeah, I understand, but we control the API on both sides. :) I guess I was suggesting that we could just change `docs.getsentry.com` to produce what we want.

Comment (dcramer) [inline src/sentry/api/endpoints/project_platform_docs.py]: Diff services, diff APIs. The wizard json is the format required for docs to generate stuff. It's easier to keep it stable this way, and we dont have to expose data we dont need.

Comment (mattrobenolt) [inline src/sentry/api/endpoints/project_platform_docs.py]: That's fair.

**Pull request merged**